### DISCUSSION TITLE
Display issues on Sonoma 14.1

### DISCUSSION THREAD

**Discussion by michaelherger** [rxhanson/RectanglePro-Community#379] [General]
I'm not sure whether this an issue introduced with Sonoma. But that's what I'm currently using anyway. Rectangle Pro is v3.0.12 (153).

I have a MBP M2Pro with an external monitor. The external monitor is a 27" 2560x1440x (not HiDPI). When I bring up the preferences, some of the icons are really small, impossible to be recognised:

<img width="299" alt="Screenshot 2023-11-03 at 11 31 47" src="https://github.com/rxhanson/RectanglePro-Community/assets/2789989/7a41b003-03d5-41db-a2b2-4c08f52e9e2e">

And there seems to be a problem with escaping the division slash `/`: as I was facing issues with absolute pixel numbers in App Layouts (sometimes Rectangle would stop resizing windows) I started using ratios instead. And instead of `0.666666` I wanted to use `2/3` - which according to the docs is valid and does work. Alas: the slash gets escaped whenever I enter the settings. And after a while it's a big mess, and the numbers wouldn't work any more:

![Screenshot 2023-11-03 at 11 25 22](https://github.com/rxhanson/RectanglePro-Community/assets/2789989/da68806a-8e5a-44e0-84df-126f03c50fad)

**Comment by rxhanson** [comment]
Thanks for reporting!

I'll slate this for the next release. The button image sizes are caused by Sonoma, but I'm not sure about the repeating slashes. Can you export your config and send it to me? There's a button in the settings tab.

**Reply by michaelherger** [reply]
Oops... I'm sorry for the noise: the backslash issue is my bug. 

You might remember my question about automatically re-configuring app layouts depending on the network. I believe you even warned me about digging too deep and how that was asking for trouble. I should not have bothered you with this issue, as it's exactly as you said: I created the trouble myself, while automatically rewriting the configuration to change triggers according to the environment... 🤦🏻.

**Reply by michaelherger** [reply]
Thanks for the update! Button display issue is now resolved.
